### PR TITLE
gh-104924: Fix `read()able` in `http.client` log messages

### PR DESCRIPTION
**Pull request by arhadthedev**
https://github.com/search?q=repo%3Apython%2Fcpython%20read()able&type=code shows no other occurrence of this typo.

* Issue: gh-104924

### CODE REVIEW

File: Lib/http/client.py [hunk 1]
```diff
@@ -1024,7 +1024,7 @@ def send(self, data):
             print("send:", repr(data))
         if hasattr(data, "read") :
             if self.debuglevel > 0:
-                print("sendIng a read()able")
+                print("sendIng a readable")
```
Comment (gaogaotiantian) [inline Lib/http/client.py]:
```suggestion
                print("sending a readable")
```

File: Lib/http/client.py [hunk 2]
```diff
@@ -1054,7 +1054,7 @@ def _output(self, s):
 
     def _read_readable(self, readable):
         if self.debuglevel > 0:
-            print("sendIng a read()able")
+            print("sendIng a readable")
```
Comment (gaogaotiantian) [inline Lib/http/client.py]:
```suggestion
            print("reading a readable")
```

**Review by gaogaotiantian — CHANGES_REQUESTED** [review]
I think `sendIng` is an obvious typo. On line 1057, from some brief reading, it's not "sending" a readable, but "reading" from a readable(as indicated by the function name).

**Review by gaogaotiantian — APPROVED** [review]

**Comment by LyingDalai** [comment]
I have looked for any related typo with “Ing the read()”. I know PyCharm can make funny replacements but I still haven’t found where this would come from.

**Review by corona10 — APPROVED** [review]
lgtm

**Pull request merged**